daffodil-io/src/main/scala/org/apache/daffodil/processors/charset/BitsCharset.scala

```diff
@@ -54,6 +51,7 @@ trait BitsCharset extends Serializable {
     case _ => false
   }
   def name: String
+  def aliases: Seq[String] = Nil
```

Comment:
See above at Line 40 - the comment needs to be updated. The motivation for BitsCharset is not the same anymore as it was. It's not about final methods, but the excessive complexity of dealing with Java charset and decoders/encoders.

I can't comment on a line that's not part of the diff, so put this comment here at the first Diff line after.

Reply:
Fixed.